lighthouse-core/computed/trace-of-tab.js

```diff
@@ -43,7 +36,20 @@ class TraceOfTab {
    * @return {Promise<LH.Artifacts.TraceOfTab>}
   */
   static async compute_(trace) {
-    return LHTraceProcessor.computeTraceOfTab(trace);
+    const traceOfTab = await LHTraceProcessor.computeTraceOfTab(trace);
+    const {timings, timestamps, firstContentfulPaintEvt} = traceOfTab;
+    const {firstContentfulPaint: firstContentfulPaintTiming} = timings;
+    const {firstContentfulPaint: firstContentfulPaintTs} = timestamps;
+    if (!firstContentfulPaintEvt || !firstContentfulPaintTiming || !firstContentfulPaintTs) {
```

Comment:
hate to do it to the line break, but I guess the last two should be checking against `undefined`?

Reply:
fair, done!